client/components/select-dropdown/index.jsx

```diff
@@ -41,7 +41,7 @@ class SelectDropdown extends Component {
 		options: PropTypes.arrayOf(
 			PropTypes.shape( {
 				value: PropTypes.string.isRequired,
-				label: TranslatableString.isRequired,
+				label: PropTypes.oneOfType( [ TranslatableString, PropTypes.node ] ).isRequired,
```

Comment:
I had to add `PropTypes.node` here because the `TranslatableString` PropType was throwing warnings in the console, even though all the labels I am sending are all translated with `translate()`. I spent a lot of time trying to debug why, but couldn't find any other way around it. If you have any ideas or insight here, let me know. Otherwise, I don't think there is any major harm adding the type `PropTypes.node` here, but if you think differently, please let me know. 🙂